ldms/python/ldmsd/ldmsd_controller

```diff
@@ -1712,13 +1712,16 @@ if __name__ == "__main__":
                                          "To connect to an ldmsd, either give " \
                                          "the socket path of the ldmsd or " \
                                          "give both hostname and inet control port. " \
-                                         "If all are given, the sockname takes the priority.")
-        parser.add_argument('--host',
+                                         "If all are given, the sockname takes the priority.",
+                                         add_help=False)
+        parser.add_argument('-H', '--help', action='help', default='==SUPPRESS==',
```

Comment:
I think '?' might be better than -H. @nichamon, what do you think?

Reply:
I'll update it to any concensus value. I think -? is somewhat less common than -H in the python world, but i can't point to a survey to prove it. Could probably add both, in fact, unless you have a collision with -H in mind for a feature down the road.

Reply:
> I think '?' might be better than -H. @nichamon, what do you think?

If we want to have a shorthand for --help, I vote for -?. I think we won't use -? for any other arguments. Many programs use -H for a particular argument. For example, docker's -H means a path to a file containing a host list. salloc's -H is equivalent to --hold.

Reply:
@baallan, Ok, I like this patch, but can we go with -??

Reply:
Will update with the ? instead of H.